### ISSUE TITLE
Request for Journal(s): Physical Review (APS)

### ISSUE THREAD

**Issue by ax3l** [sachsmc/pandoc-journal-templates#7]
American Physical Society (APS) journals (Physical Review ...), e.g.
- PRL
- PRAB (OA)

all follow these style guidelines: https://cdn.journals.aps.org/files/styleguide-pr.pdf

Authors page: https://journals.aps.org/prab/authors
REVTeX: https://journals.aps.org/revtex ([FAQ](https://journals.aps.org/revtex/revtex-faq))

**Comment by ttxtea** [comment]
Yes `revtex4-2` support for `pandoc` would be a massif step forward.

I get

~~~
! Undefined control sequence.
<argument> \collaboration@sw
~~~

when using

~~~
---
documentclass: revtex4-2
csl: american-physics-society.csl
...
~~~
